.github/workflows/ci.yaml

```diff
@@ -404,3 +402,59 @@ jobs:
 
       - name: Lint Prettier
         run: yarn lint:prettier
+
+  publish_app:
```

Comment:
@n1ru4l should I merge it with the docker things?

Reply:
its okay for now